examples/timestamping.rs

```diff
@@ -0,0 +1,93 @@
+#[macro_use]
+extern crate exonum;
+#[macro_use]
+extern crate exonum_testkit;
+extern crate serde_json;
+
+use exonum::crypto::{gen_keypair, PublicKey};
+use exonum::blockchain::{Block, Schema, Service, Transaction};
+use exonum::messages::{FromRaw, Message, RawTransaction};
+use exonum::storage::Fork;
+use exonum::encoding;
+use exonum_testkit::{ApiKind, TestKitBuilder};
+
+// Simple service implementation.
+
+const SERVICE_ID: u16 = 1;
```

Comment:
I propose to use different ids in the examples because users often create services with "reserved" ids.